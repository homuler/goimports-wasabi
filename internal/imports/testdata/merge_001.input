// merge multiple single import declarations
package main

import "fmt"
import "context"
import "strconv"

func main() {
	fmt.Println(context.Background())
	fmt.Printf("%v\n", strconv.Atoi("10"))
}
