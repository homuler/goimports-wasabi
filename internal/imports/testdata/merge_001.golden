// merge multiple single import declarations
package main

import (
	"context"
	"fmt"
	"strconv"
)

func main() {
	fmt.Println(context.Background())
	fmt.Printf("%v\n", strconv.Atoi("10"))
}
